pkg/limitador/redis_cache_storage_options_test.go

```diff
@@ -116,7 +116,7 @@ func TestRedisCachedDeploymentOptions(t *testing.T) {
 			DeploymentStorageOptions{
 				Command: []string{
 					"redis_cached",
-					"redis://example.com:6379",
+					"$(URL)",
```

Comment:
`URL` seems too generic. It is not even a env var limitador can or should understand (even though it is capable of reading it). 

What about something like `LIMTADOR_OPERATOR_REDIS_URL`?? (take into account I am and I have always been horrible picking names)

Reply:
The change of naming makes sense as its a public viewable field and always good to know where it comes from.